### PR TITLE
Router lanes

### PR DESCRIPTION
**Pull request by Karry**
Hi. 

These changes allows to add route lanes and suggested lanes to route description.
```
./Demos/Routing --debug --routeDebug \
  europe-czech-republic-20200229-004825 \
  50.0922 14.4928 \
  49.2153 16.62401

...
  7.5km  0.0km  0:08h  0:00h // at 50.05044 N 14.47787 E
                             // 0.1h 7.500853 km Way 98507071[0] => Way 98507071[1]
                             // Name: 'Jižní spojka (MO)'
                             // Type name: 'highway_motorway_trunk'
                             // Crossing: Crossing from 'Jižní spojka (MO)' to 'Jižní spojka (MO)'
                             // Direction: Turn: Straight on, 0.913341 degrees Curve: Straight on, 0.913341 degrees
                             // Lanes: 4 [oneway] : <unspecified> <unspecified> <unspecified> slight_right
                             // Suggested lanes: <3, 3>
                             // Max. speed 80km/h
```
Suggested lanes `<3, 3>` means that driver should used the most right lane from 4  (with "slight_right" sign).

Later I want to integrate these descriptions to navigation code and Qt UI...

### CODE REVIEW

File: libosmscout/src/osmscout/routing/Route.cpp [hunk 1]
```diff
@@ -19,11 +19,12 @@
 
 #include <osmscout/routing/Route.h>
 
-#include <sstream>
-
 #include <osmscout/system/Assert.h>
 #include <osmscout/system/Math.h>
 
+#include <sstream>
+#include <sstream>
```
Comment (Framstag) [inline libosmscout/src/osmscout/routing/Route.cpp:26]: Twice?

Comment (Karry) [reply libosmscout/src/osmscout/routing/Route.cpp]: amend + force push

**Comment by Framstag** [comment]
Nice addition :-) I'm interested how good lane suggestions already work? I haven't studied the algorithm in details but it look rather simple.

**Comment by Karry** [comment]
It actually is simple. Test results from desk looks good. But real testing starts when it will be available thru mobile UI and voice commands :-)

**Pull request merged**